src/System.Net.Http.WinHttpHandler/tests/System.Net.Http.WinHttpHandler.Tests.csproj

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'd like to see this revisited at a later date.  The pattern for the rest of the corefx projects AFAIK is that the unit tests link against the library.  We should be consistent within the repo as it makes making large changes (especially infrastructure related ones) a lot easier and makes it less likely that we'll make mistakes